ldk/node/src/hostClients/stoppables.ts

```diff
@@ -1,11 +1,15 @@
 /**
  * The StreamListener type is a function provided to start a stream. It accepts two parameters and returns nothing.
  *
+ * @template T - The type of the messages sent.
  * @param error - Null, unless an error is present.
  * @param input - The event that's being generated.
  */
 export type StreamListener<T> = (error: string | null, input?: T) => void;
 
+/**
+ * @template T - The type of the messages sent.
```

Comment:
Sorry, these should be `@typeParam` tags not `@template`.

Reply:
It seems to be an alias? See [here](https://typedoc.org/guides/doccomments/#typeparam-param-name) and the comment "You may also use the template tag".

`@typeParam` gives me an ESLint warning, whereas `@template` doesn't. I don't see where I can configure this either. 😕

Reply:
OH! TIL!

That `@template` is an alias for `@typeParam` is totally not intuitive at all but I'll take it!

I think you can configure whichever rule is complaining about to support additional tags in the eslint config file